src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -116,6 +117,7 @@ static SimplexResult OPT_TryToCallSimplex(
     assert(opt >= 0 && opt < 2);
     OptimizationStatistics& optimizationStatistics = problemeHebdo->optimizationStatistics[opt];
 
+    long long updateTime = 0;
```

Comment:
auto or why this variable is declared far from its assignment or usage?

Reply:
declared before the if scope where we modifiy it

Reply:
Ok sorry